packages/flutter/lib/src/material/app.dart

```diff
@@ -785,35 +785,23 @@ class MaterialApp extends StatefulWidget {
 /// discoverable, so consider adding a Scrollbar in these cases, either directly
 /// or through the [buildScrollbar] method.
 ///
-/// [MaterialScrollBehavior.androidOverscrollIndicator] specifies the
-/// overscroll indicator that is used on [TargetPlatform.android]. When null,
-/// [ThemeData.androidOverscrollIndicator] is used. If also null, the default
-/// overscroll indicator is the [GlowingOverscrollIndicator]. These properties
-/// are deprecated. In order to use the [StretchingOverscrollIndicator], use
-/// the [ThemeData.useMaterial3] flag, or override
-/// [ScrollBehavior.buildOverscrollIndicator].
+/// [ThemeData.useMaterial3] specifies the
+/// overscroll indicator that is used on [TargetPlatform.android], which
+/// defaults to true, resulting in a [StretchingOverscrollIndicator]. Setting
+/// [ThemeData.useMaterial3] to false will instead use a
+/// [GlowingOverscrollIndicator].
 ///
 /// See also:
 ///
 ///  * [ScrollBehavior], the default scrolling behavior extended by this class.
 class MaterialScrollBehavior extends ScrollBehavior {
   /// Creates a MaterialScrollBehavior that decorates [Scrollable]s with
-  /// [GlowingOverscrollIndicator]s and [Scrollbar]s based on the current
+  /// [StretchingOverscrollIndicator]s and [Scrollbar]s based on the current
   /// platform and provided [ScrollableDetails].
   ///
-  /// [MaterialScrollBehavior.androidOverscrollIndicator] specifies the
-  /// overscroll indicator that is used on [TargetPlatform.android]. When null,
-  /// [ThemeData.androidOverscrollIndicator] is used. If also null, the default
-  /// overscroll indicator is the [GlowingOverscrollIndicator].
-  const MaterialScrollBehavior({
-    @Deprecated(
-      'Use ThemeData.useMaterial3 or override ScrollBehavior.buildOverscrollIndicator. '
-      'This feature was deprecated after v2.13.0-0.0.pre.'
-    )
-    super.androidOverscrollIndicator,
-  }) : _androidOverscrollIndicator = androidOverscrollIndicator;
-
-  final AndroidOverscrollIndicator? _androidOverscrollIndicator;
+  /// For [TargetPlatform.android], if [ThemeData.useMaterial3] is true then [indicator] is a
+  /// [StretchingOverscrollIndicator], otherwise it's a [GlowingOverscrollIndicator].
```

Comment:
```suggestion
  /// For [TargetPlatform.android], if [ThemeData.useMaterial3] is true then the over scroll indicator is a
  /// [StretchingOverscrollIndicator], otherwise it's a [GlowingOverscrollIndicator].
```